<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Citas</class>
 <widget class="QWidget" name="Citas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/Secretaria.png</normaloff>:/icons/icons/Secretaria.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Registro de Consultas</string>
   </property>
  </widget>
  <widget class="QTableView" name="pacientesTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="citasTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="doctorTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonEliminar">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>310</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonGuardar">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QTableView" name="consultasGeneradasTable">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tituloTablaPacientesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista de pacientes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista de doctores:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista de citas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista de consultas generadas:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
